### ISSUE TITLE
Flake 8 errors are not printed in TravisCI

### ISSUE THREAD

**Issue by gronke** [iocage/libiocage#34]
The output of TravisCI build logs leaves a hint that there are 67 code style issues, but does not print them or cause a failed build status.

From the [Travis CI build logs](https://travis-ci.org/iocage/libiocage/builds/268842595):

```
...
$ cd iocage/libiocage
0.41s$ git fetch origin +refs/pull/33/merge:
remote: Counting objects: 45, done.
remote: Compressing objects: 100% (15/15), done.
Unpacking objects: 100% (45/45), done.
remote: Total 45 (delta 30), reused 44 (delta 30), pack-reused 0
From https://github.com/iocage/libiocage
 * branch            refs/pull/33/merge -> FETCH_HEAD
$ git checkout -qf FETCH_HEAD
0.01s$ source ~/virtualenv/python3.6/bin/activate
$ python --version
Python 3.6.2
$ pip --version
pip 9.0.1 from /home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages (python 3.6)
install
1.16s$ pip3 install flake8
Collecting flake8
  Downloading flake8-3.4.1-py2.py3-none-any.whl (68kB)
    100% |████████████████████████████████| 71kB 2.6MB/s 
Collecting pyflakes<1.6.0,>=1.5.0 (from flake8)
  Downloading pyflakes-1.5.0-py2.py3-none-any.whl (225kB)
    100% |████████████████████████████████| 225kB 3.5MB/s 
Collecting mccabe<0.7.0,>=0.6.0 (from flake8)
  Downloading mccabe-0.6.1-py2.py3-none-any.whl
Collecting pycodestyle<2.4.0,>=2.0.0 (from flake8)
  Downloading pycodestyle-2.3.1-py2.py3-none-any.whl (45kB)
    100% |████████████████████████████████| 51kB 7.8MB/s 
Installing collected packages: pyflakes, mccabe, pycodestyle, flake8
Successfully installed flake8-3.4.1 mccabe-0.6.1 pycodestyle-2.3.1 pyflakes-1.5.0
4.69s$ .travis/flake8.sh
Current Error Count: 67
usage: git rev-list [OPTION] <commit-id>... [ -- paths... ]
  limiting output:
    --max-count=<n>
    --max-age=<epoch>
    --min-age=<epoch>
    --sparse
    --no-merges
    --min-parents=<n>
    --no-min-parents
    --max-parents=<n>
    --no-max-parents
    --remove-empty
    --all
    --branches
    --tags
    --remotes
    --stdin
    --quiet
  ordering output:
    --topo-order
    --date-order
    --reverse
  formatting output:
    --parents
    --children
    --objects | --objects-edge
    --unpacked
    --header | --pretty
    --abbrev=<n> | --no-abbrev
    --abbrev-commit
    --left-right
    --count
  special purpose:
    --bisect
    --bisect-vars
    --bisect-all
fatal: No names found, cannot describe anything.
Comparing with last stable release: 
's Error Count: 67
The command ".travis/flake8.sh" exited with 0.
Done. Your build exited with 0.
```

**Comment by igalic** [comment]
a simple fix would be to tag the import commit as `0.0.0`

**Comment by skarekrow** [comment]
Yep that ^
